src/js/plugins/tts/plugin.tts.js

```diff
@@ -0,0 +1,253 @@
+/**
+ * Plugin for Text to Speech in BookReader
+ */
+
+import FestivalTTSEngine from './FestivalTTSEngine.js';
+
+// Default options for TTS
+jQuery.extend(BookReader.defaultOptions, {
+    server: 'ia600609.us.archive.org',
```

Comment:
Should we be hardcoding this address?

Reply:
This is the way it was before (for better or worse) :P